app/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
-
+    <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
They are granted here on the package itself:
https://github.com/UseAlloy/alloy-codeless-lite-android/blob/main/sdk/src/main/AndroidManifest.xml

Did you have to add them to run this example? I don't have it and runs on my android phone.